tests/Dfc.CourseDirectory.WebV2.Tests/HttpResponseMessageExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Net.Http;
+
+namespace Dfc.CourseDirectory.WebV2.Tests
+{
+    public static class HttpResponseMessageExtensions
+    {
+        public static void EnsureNotErrorStatusCode(this HttpResponseMessage response)
```

Comment:
a comment explaining the raison d'etre for this would be handy, in a github diff with no code navigation I can't entirely fathom what it's for, it seems a bit odd in isolation

Reply:
Happy if I copy message from e268946506d5e69cd5f826abdc7d4a9243262ce6 into here?

Reply:
Yep, looks good to me

Reply:
`EnsureNonErrorStatusCode` perhaps might read more nicely